[Marketplace Contribution] XSOAR File Management - Content Pack Update (#32961)
demisto/content: Packs/XSOARFileManagement/ReleaseNotes/1_1_0.md

#### Integrations

##### XSOAR File Management

> - Added additional parameter for x-xdr-auth-id for API Key in XSOAR 8

Reviewer: ```suggestion
- Added an additional parameter **XSOAR Server API Key ID** for XSOAR 8.
```

> - Modified client.delete_file() command to accept incident_id parameter for XSOAR 8 file deletion

Reviewer: ```suggestion
- Updated the  **file-management-delete-file** command to support delete file in XSOAR 8.

```
